src/transaction.ts

```diff
@@ -73,25 +74,41 @@ export interface KeyRange {
 }
 
 export interface ReadRequest extends RequestOptions {
-  keySet?: s.KeySet;
+  session?: string | null;
+  transaction?: TransactionSelector | null;
+  table?: string | null;
+  index?: string | null;
+  columns?: string[] | null;
   keys?: string[];
-  ranges?: KeyRange[];
-  index?: string;
-  columns?: string[];
-  limit?: number;
-  resumeToken?: ResumeToken;
-  partitionToken?: Uint8Array | string;
+  ranges?: spannerClient.spanner.v1.IKeyRange[];
```

Comment:
I think we had a custom `KeyRange` type that change this is stomping on.

Reply:
Create new Type